### PR TITLE
Add doc on the controlPlaneOnly field description of TraitDefinition

### PR DESCRIPTION
**Pull request by fourierr**
Signed-off-by: Xiangbo Ma <maxiangboo@cmbchina.com>
Add doc on the controlPlaneOnly field description of TraitDefinition

### CODE REVIEW

**Review by wonderflow — COMMENTED** [review]
let's hold until the feature release in v1.4

**Comment by fourierr** [comment]
> let's hold until the feature release in v1.4

👌

**Review by wonderflow — APPROVED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.3/platform-engineers/oam/x-definition.md [hunk 1]
```diff
@@ -212,7 +213,7 @@ spec:
 * 是否由该运维特征管理工作负载（`.spec.manageWorkload`）,可缺省字段，bool 类型，设置为 true 则标识这个运维特征会负责工作负载的创建、更新、以及资源回收，通常是灰度发布的运维特征会具备这个能力。
 * 该运维特征是否不计入版本变化的计算（`.spec.skipRevisionAffect`）,可缺省字段，bool 类型，设置为 true 则标识这个运维特征的修改不计入版本的变化，即用户在应用中纯粹修改这个运维特征的字段不会触发应用本身的版本变化。
 * 运维能力是否感知组件版本的变化（`.spec.revisionEnabled`）字段，可缺省字段，bool 类型，设置为 true 表示组件会生成的资源后缀会带版本后缀。
-
+* 运维能力产生的资源是否部署到管控集群（`.spec.controlPlaneOnly`）字段，可缺省字段，bool 类型，设置为 true 表示 trait 生成的资源会被部署到管控集群，即`local`
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.3/platform-engineers/oam/x-definition.md]: please remove the change in v1.3, we don't  have thisi feature in v1.3

Comment (fourierr) [reply i18n/zh/docusaurus-plugin-content-docs/version-v1.3/platform-engineers/oam/x-definition.md]: ok

**Review by wonderflow — APPROVED** [review]
thanks

**Pull request merged**